schema/sales.js

```diff
@@ -1,44 +1,12 @@
 import gravity from '../lib/loaders/gravity';
 import Sale from './sale';
+import SaleSorts from './sorts/sale_sorts';
```

Comment:
I think it'd make more sense, organizationally if this was in /sale/sorts.js and sale.js => sale/index.js